lighthouse-core/lib/url-shim.js

```diff
@@ -17,6 +17,24 @@ const Util = require('../report/html/renderer/util.js');
 const URL = /** @type {!Window["URL"]} */ (typeof self !== 'undefined' && self.URL) ||
     require('url').URL;
 
+const tldPlusOne = {
```

Comment:
let's hardcode just the keys of this list (and add .co.uk)

add comment saying that https://publicsuffix.org/ is the real source of this data. and hypothetically we could pull in a dep like https://npms.io/search?q=public+suffix+list to do this forreal, but at significant byte cost.

and admit we're taking the lazy way out.

Reply:
shared a sheet with you on the most popular 2-level TLDs, but summary: if we did a regex with /\.(co|com|org|edu|gov)\.TLD$/ that should cover most cases

Reply:
I added a new list to the regex coming from http archive. Not sure how I should comment it in the code. Also i'm not matching all 1000 entries